FROM python:3
RUN apt-get update

